meinberlin/apps/plans/templates/meinberlin_plans/includes/plan_form.html

```diff
@@ -44,9 +44,9 @@ <h2 class="form__subheading">{% translate 'Contact for questions' %}</h2>
     {% include 'a4forms/includes/form_field.html' with field=form.contact_phone %}
     {% include 'a4forms/includes/form_field.html' with field=form.contact_url %}
 
-    <h2 class="form__subheading">{% translate 'Presentation in Project Overview' %}</h2>
+    <h2 class="form__subheading">{% translate 'Presentation in Kiezradar' %}</h2>
     <p class="form-hint form-hint--margin">
-        {% translate 'The following information affects the presentation of the project in the general project overview.' %}
+        {% translate 'The following information affects the presentation of the project in the general Kiezradar.' %}
```

Comment:
```suggestion
        {% translate 'The following information affects the presentation of the project in the Kiezradar.' %}
```
I think the word general doesn't really make a lot of sense here, right?